submissions/ASTkach/js-world/index.js

```diff
@@ -0,0 +1,83 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+    species: "dog",
+    name: "Elvis",
+    gender: "male",
+    legs: 4,
+    hands: 0,
+    saying: "wooffle-wooffle",
+    friends: ["Franke", " Marilyn", " Audrey"],
+};
+
+const cat = {
+    species: "cat",
+    name: "Audrey",
+    gender: "female",
+    legs: 4,
+    hands: 0,
+    saying: "hiss-hiss",
+    friends: "",
+};
+
+const woman = {
+    species: "human",
+    name: "Marilyn",
+    gender: "female",
+    legs: 2,
+    hands: 2,
+    saying: "Well it's a good day for singing a song",
+    friends: [" Franke", " Audrey"],
+};
+
+const man = {
+    species: "human",
+    name: "Franke",
+    gender: "male",
+    legs: 1,
+    hands: 2,
+    saying: "Honey, where's my wooden leg?",
+    friends: [" Elvis", " Marilyn"],
+};
+
+let jsWorld = [dog, cat, woman, man];
+
+let result = jsWorld.map((item) => {
+    return item.species + "; " + 
+           item.name + "; " + 
+           item.gender + "; " + 
+           item.legs + "; " + 
+           item.hands + "; " + 
+           item.saying + "; " + 
+           item.friends;
```

Comment:
Too much repetition of code (`item.`, `+"; " +`).
You will benefit from `Array#map` and `Array#join` to make code dry.